### DISCUSSION TITLE
Can you assign two icons to Amarok?

### DISCUSSION THREAD

**Discussion by jcperil** [Arcticons-Team/Arcticons#1686] [Q&A] [answered]
[Amarok](https://github.com/deltazefiro/Amarok-Hider) is an Android application which enables you to hide your private files & apps with a single click.

By default, it has the following icon:
<p align=center><img src="https://github.com/Donnnno/Arcticons/assets/89805719/d923bfd1-96c9-4dd5-8cf7-8211d45da2ec" width="125px"/></>

Recently, it pushed an update with a new feature where you can disguise Amarok as a calendar app. If you enable that, the app icon changes to that of a calendar.

<p align=center><img src="https://github.com/Donnnno/Arcticons/assets/89805719/924fc472-52f8-43bf-b920-f02482066b81" width="125px"/><p/>

It is kind of an odd situation, hence my odd request: Is it possible to assign both icons to Amarok?

**Comment by Donnnno** [comment]
Yes, that should be possible, are the activities different when you gather them using icon request?

**Reply by jcperil** [reply]
Yes, the default is

<resources>

	
	<item component="ComponentInfo{deltazero.amarok.foss/deltazero.amarok.launcher.default}" drawable="amarok" />

</resources>

When disguised as calendar app, the activity is
<resources>

	
	<item component="ComponentInfo{deltazero.amarok.foss/deltazero.amarok.launcher.calendar}" drawable="calendar" />

</resources>

**Reply by Donnnno** [reply, marked as answer]
Done!
